src/App.tsx

```diff
@@ -1,26 +1,214 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import * as todoService from './api/todos';
+import { ErrorMessage } from './types/ErrorMessage';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | ''>('');
+  const [filterValue, setFilterValue] = useState<Filter>(Filter.All);
+  const [todosInTheBoot, setTodosInTheBoot] = useState<number[]>([]); //to make sure
```

Comment:
maybe `loadingTodoIds` - to show it contains ids, not todos
boot might be confusing here 🙂 

>//to make sure

Make sure what?) I'd delete this comment